src/App.php

```diff
@@ -1077,10 +1077,10 @@ public function runFrontend()
 
 		// in install mode, any url loads install module
 		// but we need "view" module for stylesheet
-		if ($this->getMode()->isInstall() && $this->module != 'view') {
-			$this->module = 'install';
-		} elseif (!$this->getMode()->has(App\Mode::MAINTENANCEDISABLED) && $this->module != 'view') {
-			$this->module = 'maintenance';
+		if ($this->getMode()->isInstall() && $this->module !== 'install') {
+			$this->internalRedirect('install');
```

Comment:
Why add a redirect here where there wasn't?

Reply:
This is a bug fix.

Currently we won't reach the installer because the new routing doesn't check the module, but on the other hand the module doesn't load the new Module anymore.